vignettes/basic_usage.Rmd

```diff
@@ -62,22 +62,27 @@ changing soon).
 ```{r}
 # Set the number of tokens to pad/truncate the result to.
 n_tokens <- 20L
-n_inputs <- length(text_inputs)
-tokenized <- torchtransformers::tokenize_bert(text_inputs$segment1,
-                                              text_inputs$segment2,
-                                              n_tokens = n_tokens)
+tokenized <- torchtransformers::tokenize_bert(
```

Comment:
I don't like just using the default here, even though they happen to be correct for the model used. Can we explicitly set the tokenizer/vocab, with a comment that we have to use the right vocab to be compatible with the uncased bert model?

Reply:
Ha, sure! You wrote that part, I just changed the indentation 🙃

Reply:
I know. :) When I wrote it, there wasn't an option for specifying the tokenizer--it was hard-coded in {tt}. That realization is partly what led to this whole set of PRs. :-D

Reply:
I'll write a story about handling this a little better. In theory tidy_bert_output should be able to deal with this using the original, untokenized data (or we can create that data as a tt dataset, which will become tokenized in the model process).

Reply:
or maybe it was just hard-coded in the dataset constructor? I don't remember anymore. :-D